### PR TITLE
Chal18

### PR DESCRIPTION
**Pull request by logic-legolas**
I finished challenge 18 by fixing the infinite loop challenge

### CODE REVIEW

**Comment by logic-legolas** [comment]
@rickyc
@grizellong

**Comment by logic-legolas** [comment]
@grizelong

**Comment by logic-legolas** [comment]
@gizzelong

**Comment by rickyc** [comment]
lol none of those were right :( @griselleong

**Comment by logic-legolas** [comment]
Thanks ricky

File: chal18.html [hunk 1]
```diff
@@ -0,0 +1,27 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="utf-8">
+    <title></title>
+  </head>
+  <body>
+    <script type="text/javascript">
+      let numbers = [Math.random(),Math.random(),Math.random(),Math.random(),Math.random()];
```
Comment (rickyc) [inline chal18.html:9]: `const` since you are never assigning the variable to something else.

File: chal18.html [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+      console.log(numbers);
+      function sorter() {
+        for (var foo = 0; foo < numbers.length; foo++) {
```
Comment (rickyc) [inline chal18.html:12]: These should be `let` and I would name the variables better. I would go with `i`, `j` in this case.

File: chal18.html [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+          for (var i = 0; i < numbers.length; i++) {
+            if (numbers[i]>numbers[i+1]){
+              var n = numbers[i+1];
```
Comment (rickyc) [inline chal18.html:15]: `const` since you aren't reassigning the value.

**Review by rickyc — APPROVED** [review]
Looks good, just a few changes in terms of ES6 syntax to be made.

**Comment by rickyc** [comment]
Merge this in after you make the updates.

**Pull request merged**